subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -742,6 +742,12 @@ Use `implementation.bundle(libs.bundles.testing)` instead.
 
 For more information, see the updated <<jvm_test_suite_plugin.adoc#sec:declare_an_additional_test_suite, declare an additional test suite>> example in the JVM Test Suite Plugin section of the user guide and the link:{groovyDslPath}/org.gradle.api.artifacts.dsl.DependencyAdder.html[`DependencyAdder`] page in the DSL reference.
 
+==== Incremental compilation temporarily changes the output location
+
+Java and Groovy compilation changes the compiler output location. This might affect some annotation processors that allow users to wire some action to file paths (e.g. `-XepExcludedPaths` in Error Prone).
```

Comment:
```suggestion
Java and Groovy compilation changes the compiler output location.
This might affect some annotation processors that allow users to wire some action to file paths (e.g. `-XepExcludedPaths` in Error Prone).
```

❌ Same here, one sentence per line, please.